tests/server/3-document.spec.js

```diff
@@ -208,5 +191,19 @@ describe('Document', () => {
               res.body.message.should.equal('Document not found');
               done();
           });
-    })
-})
+    });
+    
+    it('should return all regular documents if role=regular query is used', (done) => {
+        server
+          .get('/api/documents?role=regular')
+          .set('authorization', userToken)
+          .end((err, res) => {
+              console.log(res.body);
+              res.status.should.equal(200);
+              for(let i=0; i<res.body.length; i++) {
```

Comment:
### Critical

`res.body` is an array, it's more appropriate to use the forEach API to loop through the collection.